### ISSUE TITLE
Change Buffergator key mappings

### ISSUE THREAD

**Issue by daveroberts** [jeetsukumaran/vim-buffergator#19]
I'd like to change some of buffergator's key mappings in my vimrc file.

I use ijkl for navigation instead of hjkl.

As a result, when I press 'i', I want to move up, not open the selected buffer.  Can I change this?

**Comment by michaelfresco** [comment]
Yes it sucks that the keymappings cannot be changed. It's such a nice plugin.

**Comment by christian-oudard** [comment]
@michaelfresco We're asking about the movement keys once buffergator is open, not the key to open buffergator in the first place. I have already done a remapping similar to what you're saying.
